### ISSUE TITLE
a11y: Review Dropdown Accessibility

### ISSUE THREAD

**Issue by StephenRiosDev** [AlaskaAirlines/auro-formkit#455]
### Description

Assess accessibility of the dropdown.

W3C example: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/

Criteria:
- Trigger button should tell the user that a new dropdown will open
- Dropdown should be focused on open
- Dropdown should have keyboard controls for closing it (esc) and the user should be informed (e.g. press esc to close dropdown)
- Trigger button should be re-focused on close

### Exit criteria (To be determined by the Auro team)

Dropdown should meet all criteria and meet or exceed all other W3C WCAG 2.0 AA criteria.

**Comment by blackfalcon** [comment]
The floor-wide standard is to meet or exceed WCAG 2.2 Level AA. Please see [Alaska Airlines Digital Accessibility and Inclusion Policy](https://wiki.devtools.teamaag.com/standards/accessibility/digital-accessibility-policy) for more information.

**Comment by sun-mota** [comment]
No a11y violation found in layoutsupport
